src/main/java/com/reservation/useraccess/infrastructure/persistence/jpa/SystemUserMapper.java

```diff
@@ -0,0 +1,24 @@
+package com.reservation.useraccess.infrastructure.persistence.jpa;
+
+import com.reservation.useraccess.domain.SystemUser;
+import org.springframework.stereotype.Component;
+
+@Component
+class SystemUserMapper {
```

Comment:
Same metody statyczne - spring do tego nie potrzebuje bina - można zmienić na adnotacje @UtilityClass
```suggestion
@UtilityClass
class SystemUserMapper {
```